src/components/Loader/Loader.scss

```diff
@@ -0,0 +1,25 @@
+.Loader {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+
+  &__content {
+    border-radius: 50%;
+    width: 2em;
+    height: 2em;
+    margin: 1em auto;
+    border: 0.3em solid #ddd;
+    border-left-color: #000;
```

Comment:
Consider using a more descriptive variable name for the border left color, such as $border-color-active, to improve code readability.